aws_embedded_metrics/environment/default_environment.py

```diff
@@ -39,7 +41,14 @@ def get_log_group_name(self) -> str:
     def configure_context(self, context: MetricsContext) -> None:
         pass
 
-    def get_sink(self) -> AgentSink:
+    def get_sink(self) -> Sink:
         if self.sink is None:
             self.sink = AgentSink(self.get_log_group_name(), Config.log_stream_name)
         return self.sink
+
+
+class DefaultStdoutEnvironment(DefaultEnvironment):
```

Comment:
In the other implementations, we provide a `LocalEnvironment` for this purpose. Would that meet your needs here? Any particular reason to depart from that pattern?

https://github.com/awslabs/aws-embedded-metrics-node/blob/master/src/environment/LocalEnvironment.ts#L58

Reply:
Oh, I had not seen that (my need is python-only). My thinking was to maximize code re-use as DefaultEnvironment is almost exactly what was needed. I can change it to call it LocalEnvironment to be consistent though :+1:

Reply:
So made this change and now the test is happy as well :)